docs/changelog/86757.yaml

```
@@ -0,0 +1,5 @@
+pr: 86757
+summary: "[Transforms] prefer secondary auth headers for transforms"
```

```suggestion
summary: "Prefer secondary auth headers for transforms"
```

The script that creates these strips out the `[ML]` prefix, but not `[Transforms]` for some reason. But I am sure we don't want either as a prefix in the final docs end users see.